### ISSUE TITLE
QDlib in Ubuntu/Debian

### ISSUE THREAD

**Issue by berceanu** [janverschelde/PHCpack#10]
At point 2 of the **PHCpy**  [documentation](http://homepages.math.uic.edu/~jan/phcpydoc.pdf), it is specified that one has to compile **libQD** from source with the **-fPIC** flag. In the case of Ubuntu/Debian, there is a [package](https://packages.ubuntu.com/xenial/libqd-dev) providing **libQD v.2.3.11**, can one just install that instead of compiling from source? Also, what version **of libQD** does **PHCpy** 0.6.9 depend on?

**Comment by berceanu** [comment]
Apparently there is a problem with using the **QD** library from **Ubuntu 16.04** repositories. After setting `MAKEFILE = makefile_unix` in `src/Objects/makefile`, `make phc` fails with the error
`g++: error: /usr/local/lib/libqd.a: No such file or directory`
as the file `libqd.a` is located in `/usr/lib/libqd.a`, and not  `/usr/local/lib/libqd.a`

**Comment by janverschelde** [comment]
The first version of qdlib used was already quite mature, version 2.3.09 if I recall correctly.
I noticed no differences in using 2.3.14 and 2.3.17.
Thanks for noticing the differences in the install locations on Ubuntu.

**Comment by berceanu** [comment]
Yes, I wonder if the Ubuntu package is compiled with `-fPIC` though. Perhaps I should just install qdlib from source?

**Comment by C-H-Chien** [comment]
I also encounter the problem mentioned by @berceanu, where `make phc` fails with the error
`g++: error: /usr/local/lib/libqd.a: No such file or directory`. How do you solve the problem? Thanks in advance!

**Comment by janverschelde** [comment]
Try to type 'make phc_noqd' which does not need the qd library.
